spacelift/resource_gitlab_integration_test.go

```diff
@@ -1,6 +1,7 @@
 package spacelift
 
 import (
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs/vcs"
```

Comment:
Nit: move this import next to the other `github.com/spacelift-io/terraform-provider-spacelift` import